docs/content/any/project/old-docs.md

```diff
@@ -0,0 +1,7 @@
+---
+title: 0.15 Documentation
```

Comment:
Thats neat -- this is so hugo doesn't complain about broken links?

Reply:
for this somehow yeah